Content.Server/Materials/ProduceMaterialExtractorSystem.cs

```diff
@@ -41,8 +64,22 @@ private void OnInteractUsing(Entity<ProduceMaterialExtractorComponent> ent, ref
             .Sum(r => r.Quantity.Float());
         _materialStorage.TryChangeMaterialAmount(ent, ent.Comp.ExtractedMaterial, (int) matAmount);
 
-        _audio.PlayPvs(ent.Comp.ExtractSound, ent);
-        QueueDel(args.Used);
-        args.Handled = true;
+        QueueDel(used);
```

Comment:
Do I need to do any additional cleanup other than `QueueDel` when the item is in a container rather than in the hand? It seems to work fine like this in my testing, but I figured I'd ask.